concepts: output: style updates (#1487)
fluent/fluent-bit-docs: concepts/data-pipeline/output.md

---

> description: 'Destinations for your data: databases, cloud services, and more!'

Reviewer: ```suggestion
description: Learn about destinations for your data, such as databases and cloud services.
```

---

# Output

The output interface lets you define destinations for your data. Common destinations

> are remote services, local file systems, or other standard interface. Outputs are

Reviewer: ```suggestion
are remote services, local file systems, or other standard interfaces. Outputs are
```
